src/test/java/com/puppycrawl/tools/checkstyle/utils/JavadocUtilTest.java

```diff
@@ -379,6 +389,64 @@ public void testEscapeAllControlChars() {
                 JavadocUtil.escapeAllControlChars("1\\r2\\n3\\t"), "invalid result");
     }
 
+    @Test
+    public void testGetContentOfInlineCustomTag() throws Exception {
```

Comment:
Can we just use multiple input files and break this into multiple test methods or use a parameterized test? To me, in most cases, a test should fail for one reason only. This would also help with complicated traversal of AST.

Reply:
Spilt it into two tests, please take a look

Reply:
Why can't we have four, each with one expected, one actual, and one assertion?

Reply:
Done